### ISSUE TITLE
Update Jessie/Kali Repo Public Key Freeze installation.......

### ISSUE THREAD

**Issue by Siddharth0kumar** [Screetsec/TheFatRat#725]
![IMG20220712174126](https://user-images.githubusercontent.com/89460704/178487036-80d1711e-7067-408f-a939-cb6b7675dd4c.jpg)
Sir my installation process freezed there. I don't know why it's freezed.......

Please sir help me and tell me The way to delete TheFatRat tool from his external path. I mean how can i delete completely this tool....

**Comment by realvoidgojo** [comment]
Clone This repo "https://github.com/Morsmalleo/Kali-Jessie-Keys"
copy and then paste the key files to appropriate location manually /etc/apt/trusted.gpg.d 
after all this try to install thefatrat

**Comment by realvoidgojo** [comment]
I Found a Solution The Jessie repo is not in ur source.list
paste this repo deb http://ftp.debian.org/debian jessie main
in /etc/apt/source.list

**Comment by Vector1199** [comment]
im also facing same issue also added deb http://ftp.debian.org/debian jessie main this repo but still stuck at Jessie/Kali Repo Public Key point and fatrat automatically removing my sources.list and added something different sources list when I try to apt update is slows dows

**Comment by Morsmalleo** [comment]
Try this script I made for installing the correct versions mingw from **Debian Jessie**, then try the fatrat again if, the process still freezes and doesn't skip because mingw is installed from Jessie, then I'll contact Peterpt about his Repokey because that seems to be where the problem is

https://GitHub.com/Morsmalleo/TheFatRat-MinGW-fix

**Comment by Morsmalleo** [comment]
> Clone This repo "https://github.com/Morsmalleo/Kali-Jessie-Keys" copy and then paste the key files to appropriate location manually /etc/apt/trusted.gpg.d after all this try to install thefatrat

Didn't even know anyone used these glad I made a backup of them

**Comment by Morsmalleo** [comment]
I'm going to test in 5 minutes, if I'm facing the same problem then I'll contact @Peterpt because this is a major bug in the setup files bash coding if I do

**Comment by Vector1199** [comment]
@Morsmalleo I did what you said everything I thoroughly uninstalled mingw64 then  TheFatRat then as you said I installed your repo https://GitHub.com/Morsmalleo/TheFatRat-MinGW-fix then I again installed TheFatRat gave the permission and your script worked as  well it downloaded the correct version of MinGW but it still stuck at Jessie/kali repo public key 

![Screenshot_2022-07-31_04_48_29](https://user-images.githubusercontent.com/30725702/182018434-767ba7d4-495b-402b-bd1f-8a57aa4a57b5.png)

**Comment by Vector1199** [comment]
and also fatrat automatically changing sources repo which is slows down internet download speed

**Comment by Vector1199** [comment]
@Morsmalleo  any update?

**Comment by Morsmalleo** [comment]
> @Morsmalleo I did what you said everything I thoroughly uninstalled mingw64 then TheFatRat then as you said I installed your repo https://GitHub.com/Morsmalleo/TheFatRat-MinGW-fix then I again installed TheFatRat gave the permission and your script worked as well it downloaded the correct version of MinGW but it still stuck at Jessie/kali repo public key
> 
> ![Screenshot_2022-07-31_04_48_29](https://user-images.githubusercontent.com/30725702/182018434-767ba7d4-495b-402b-bd1f-8a57aa4a57b5.png)

Ok so that lets me know that it's definitely a problem I the setup file, because the script works for me to, however I've tested and tested and tested, and the Kali/Jessie repo public key always freezes. I will open an issue about this and contact TheFatRat maintainer @Peterpt

**Comment by Morsmalleo** [comment]
@Peterpt major bug in the Repokey, Kali/Jessie always freezes regardless of the OS it's running on, tested for 6 hours on multiple OS's yesterday to make sure the problem was true and it certainly is. Everytime The setup file is run, it always freezes at the `Kali/Jessie Repo Public Key` process. Regardless of the Operating System it's running on.

**Comment by Morsmalleo** [comment]
> and also fatrat automatically changing sources repo which is slows down internet download speed

Yes TheFatRat does this to install mingw from Jessie which was working until a bug hit the coding of the setup file, what fatrat does is it backs up your original sources.list file and creates its own in order to install mingw, once it's done TheFatRat sources.list file is removed and you original is reactivated

**Comment by jaaaafarsh** [comment]
@Morsmalleo any update??

**Comment by jaaaafarsh** [comment]
So anybody know what we should do now??

**Comment by Morsmalleo** [comment]
> So anybody know what we should do now??

Wait til it's fixed

**Comment by jaaaafarsh** [comment]
I tried installing it again and it worked I don't know why

**Comment by Vector1199** [comment]
I guess it is a kali repo issue I found so many bugs in kali 2022.2 pythons broken dependencies I configured manually after that most of the tools worked correctly after that I removed kali and installed Garuda Arch as the main OS and configured with Black Arch repo I searched the fat rat and it is available in the black arch repo but I don't know it is authentic or not 😕

**Comment by mekam07** [comment]
i cant install fatrat in my 32bit kali

**Comment by mekam07** [comment]
/setup.sh: line 933: /usr/local/sbin/aapt: cannot execute binary file: Exec format error  
and
[ ? ] Update Jessie/Kali Repo Public Key
